.travis.yml

```diff
@@ -128,7 +129,7 @@ install:
         fi
         source ~/virtualenv/python${pv}/bin/activate
         # explicitly install all needed python modules to cache them
-        for p in '-r requirements.txt' '-r requirements.dev.txt'; do
+        for p in '-r requirements.txt' '-r requirements.dev.txt' 'codacy-coverage coveralls'; do
```

Comment:
Shouldn't this go in `requirements.dev.txt` (that's a big part of the point of this PR).

Reply:
No, I just forgot to submit the feedback.